src/routing/FormFooter.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export default function FormFooter() {
+  // Render default form footer content
+  return (
+    <div className="row">
+      <div className="usa-width-two-thirds medium-8 columns">
+        <div className="help-footer-box">
+          <h2 className="help-heading">Need help?</h2>
+
+          <div>
+            <p className="help-talk">
+              Call us at 888-888-8888. We’re here Monday through Friday, 8:00
```

Comment:
This number can be updated to 800-827-1000. It can also be wrapped in an `<a>` tag. 

[Here's the design.va.gov](https://design.va.gov/content-style-guide/dates-and-numbers#phone-numbers) style guide for phone numbers.